addons/l10n_br/data/account_tax_template_data.xml

```diff
@@ -1,3137 +1,4427 @@
-	            }),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'tax',
-	                'account_id': ref('account_template_101050502'),
-	                'minus_report_line_ids': [ref('tax_report_ipi_2')],
-	            }),
-	        ]"/>
-	        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'base',
-	                'plus_report_line_ids': [ref('tax_report_ipi_1')],
-	            }),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'tax',
-	                'account_id': ref('account_template_201010301'),
-	                'plus_report_line_ids': [ref('tax_report_ipi_2')],
-	            }),
-	        ]"/>
-       	</record>
-
-		<record id="tax_template_in_ipi20" model="account.tax.template">
-			<field name="description">IPI 20%</field>
-			<field name="name">IPI Entrada 20%</field>
-			<field name="amount">20</field>
-			<field name="type_tax_use">purchase</field>
-			<field eval="0" name="price_include"/>
-			<field eval="0" name="tax_discount"/>
-			<field ref="l10n_br_account_chart_template" name="chart_template_id"/>
-			<field name="tax_group_id" ref="tax_group_ipi_20"/>
-			<field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'base',
-	                'minus_report_line_ids': [ref('tax_report_ipi_1')],
-	            }),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'tax',
-	                'account_id': ref('account_template_101050502'),
-	                'minus_report_line_ids': [ref('tax_report_ipi_2')],
-	            }),
-	        ]"/>
-	        <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'base',
-	                'plus_report_line_ids': [ref('tax_report_ipi_1')],
-	            }),
-	            (0,0, {
-	                'factor_percent': 100,
-	                'repartition_type': 'tax',
-	                'account_id': ref('account_template_201010301'),
-	                'plus_report_line_ids': [ref('tax_report_ipi_2')],
-	            }),
-	        ]"/>
-        </record>
-
-		<record id="tax_template_in_ipi22" model="account.tax.template">
-			<field name="description">IPI 22%</field>
-			<field name="name">IPI Entrada 22%</field>
-			<field name="amount">22</field>
-			<field name="type_tax_use">purchase</field>
-			<field eval="0" name="price_include"/>
-			<field eval="0" name="tax_discount"/>
-			<field ref="l10n_br_account_chart_template" name="chart_template_id"/>
-			<field name="tax_group_id" ref="tax_group_ipi_22"/>
-			<field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
-	            (0,0, {
-	  
```

Comment:
try_loading (can be shortened now) and this could be put in a separate file as it is quite long already